tutorcelery/patches/caddyfile

```diff
@@ -0,0 +1,6 @@
+{% if CELERY_FLOWER and CELERY_FLOWER_PUBLIC %}
+# flower
+{{ CELERY_FLOWER_HOST }}{$default_site_port} {
```

Comment:
Does flower allow to configure the Basic auth?

Reply:
Basic auth is already configured